docs/authentication.md

```diff
@@ -0,0 +1,77 @@
+# Authentication
+
+This document details how to configure your Helios installation to authenticate
+requests between the Helios CLI and the Helios masters/cluster.
+
+# TODOs for this document
+
+- configuration needed to setup master to require authentication
```

Comment:
How about starting the master with `--auth` and if no `--auth-plugin <path/to/jar>` is provided, it uses crtauth?